docs/source/ops/pruning.rst

```diff
@@ -73,9 +73,9 @@ Professional advice on database administration is strongly recommended that woul
 Determine a Suitable Pruning Offset
 -----------------------------------
 
-The :ref:`Transaction Service <transaction-service>` and the :ref:`Active Contract Service <active-contract-service>` provide offsets of the ledger end of the Transactions, and of Active Contracts snapshots respectively. Such offsets can be passed unchanged to `prune` calls, as long as they are lexicographically lower than the current ledger end.
+The :ref:`Transaction Service <transaction-service>` and the :ref:`Active Contract Service <active-contract-service>` provide offsets of the ledger end of the Transactions, and of Active Contracts snapshots respectively. Such offsets can be passed unchanged to `prune` calls, as long as they are lexicographically lower than the current ledger end. An additional constraint imposed by Canton (subject to future improvements) is that the participant you are pruning must have already exchanged the ACS commitments with other participants for the offset that you prune at.
```

Comment:
Not sure how helpful it is to mention that. But it could be setting some unnecessary expectations.
```suggestion
The :ref:`Transaction Service <transaction-service>` and the :ref:`Active Contract Service <active-contract-service>` provide offsets of the ledger end of the Transactions, and of Active Contracts snapshots respectively. Such offsets can be passed unchanged to `prune` calls, as long as they are lexicographically lower than the current ledger end. An additional constraint imposed by Canton is that the participant you are pruning must have already exchanged the ACS commitments with other participants for the offset that you prune at.
```

Reply:
I think we are over-detailing the acs commitment stuff without saying where should the user look to understand better. I would actually reduce the explanation and just reference a Canton docs page (@oliverse-da do we have pruning ops page mentioning the safe to prune offset for Canton)?

Reply:
We have this: 
> [Current Limitations](https://www.canton.io/docs/dev/user-manual/usermanual/pruning.html#current-limitations)
> ..
> Pruning of participants requires the participant to have received a commitment from each counter-participant with which it shares a contract. If a participant becomes defunct and stops sending commitments, pruning of the participant will not work. Therefore, before you disable a participant, please make sure that is not involved in any contract. This is a current limitation.

Reply:
Thanks for digging up this page @pbatko-da. That is indeed the only thing we state about participant commitments on the canton pruning page.

We used to have a whole separate section on pruning in test-environments where users used to be confronted with issues pruning "more current state", but I removed that as part of switching customers to automatic background pruning.

I am fine either way with leaving the detail here and / or add a link to this pruning page.

Reply:
FYI Added a link to https://www.canton.io/docs/dev/user-manual/usermanual/pruning.html#current-limitations

Reply:
alright, since this is not documented elsewhere, fine to leave this as is wrt to the description of the ACS commitments then. I would still add a link to that page that you found 👍

Reply:
Not sure I am up on the latest regarding documentation links - shouldn't the link point to https://docs.daml.com/ops/pruning.html rather than the canton-io page?

Reply:
Correct. And relative too.

Reply:
What I'm editing in this PR is the same page so I think there's no point to link to itself. Will remove the link

Reply:
I made the link to be `<../canton/usermanual/pruning.html>` which I hope will point to https://docs.daml.com/canton/usermanual/pruning.html when docs are released. (Is there an easy way to test it locally? I guess not)